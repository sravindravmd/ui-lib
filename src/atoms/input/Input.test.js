import React from 'react'
import { Input } from './'
import { render, cleanup } from '@testing-library/react'
import 'regenerator-runtime/runtime'

afterEach(cleanup)

const props = {
  className: 'primary',
  label: 'Field Label',
  inputProps: {
    placeholder: 'Text',
  },
}

describe('<Input/>', () => {
  it('renders Input component', () => {
    const { container } = render(<Input {...props} />)
    expect(container).toMatchSnapshot()
  })
})
