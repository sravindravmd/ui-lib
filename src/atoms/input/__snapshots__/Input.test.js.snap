// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input/> renders Input component 1`] = `
<div>
  <div
    class="sc-bdfBwQ YkLZM primary"
  >
    <div
      class="sc-gsTCUz fDhWiz"
    >
      <label
        class="input_parent"
      >
        <span
          class="input__label"
        >
          Field Label
        </span>
        <div
          class="input__wrapper"
        >
          <input
            class="sc-dlfnbm juDBAJ"
            placeholder="Text"
            value=""
          />
        </div>
      </label>
    </div>
  </div>
</div>
`;
